content/python/concepts/built-in-functions/terms/help/help.md

```diff
@@ -0,0 +1,92 @@
+---
+Title: 'help()'
+Description: 'Display the documentation of the object passed as argument.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `help()` function accepts an object as a parameter, which is a module, function, class or keyword. After being called, the documentation of the requested object is displayed.
+If no argument is passed the interactive help utility starts up on the console.
+
+## Syntax
+
+The object is passed to the `help()` function as parameter:
+
+```pseudo
+help(object)
+```
+
+To start the interactive help utility, the function is called without argument:
+
+```pseudo
+help()
+```
+  
+## Example
+
+Below is an example where a build-in function is passed as argument:
+
+```py
+help(print)
+```
```

Comment:
There need to be line breaks before and after code blocks

Reply:
Should I commit a new version with the correct spacing right now or do I have to wait for the second review?

Reply:
Go ahead and commit a new version.